scenarios/plaintext.benchmarks.yml

```diff
@@ -92,6 +92,46 @@ scenarios:
         presetHeaders: plaintext
         path: /plaintext
         pipeline: 16
+  
+  plainTextSansFilter:
+    application:
+      job: mapaction
+    load:
+      job: wrk
+      variables:
+        presetHeaders: plaintext
+        path: /hello/TesterName/42/Internet
+        pipeline: 16
+
+  plainTextEmptyFilter:
+    application:
+      job: mapaction
+    load:
+      job: wrk
+      variables:
+        presetHeaders: plaintext
+        path: /helloEmptyFilter/TesterName/42/Internet
+        pipeline: 16
+  
+  plainTextNoParamsEmptyFilter:
```

Comment:
Is the non-filter variant of this is just the `mapaction` scenario?

Reply:
Yep, it's just a call to the `Plaintext` endpoint that spits out 'Hello world'